### ISSUE TITLE
** (wpg:17272): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-SztxGTFk2U: Connection refused

### ISSUE THREAD

**Issue by Iambecomeroot** [deviantfero/wpgtk#60]
When I run any wpg command (`bash ~/.wallpapers/wp_init.sh` or `(wpg -t &)` or even `wpg -v`), I get an error similar to the following:
```

** (wpg:29160): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-SztxGTFk2U: Connection refused
```

All the commands seem to work fine, it's just kinda annoying to have that error any time I 

wpg: 4.6.0
wal: 0.6.9

**Comment by deviantfero** [comment]
seems to be an issue that has to do with this: https://bugs.launchpad.net/ubuntu/+source/at-spi2-core/+bug/1193236

anyway, as a temporary fix you could try doing `(wpg 2>/dev/null -t &)`  or create an alias for `wpg`

also take a look at this: https://bbs.archlinux.org/viewtopic.php?id=228894
and this: https://askubuntu.com/questions/432604/couldnt-connect-to-accessibility-bus

this will help you ignore those errors, I'm closing this issue as it has nothing to do with `wpgtk` itself

**Comment by Iambecomeroot** [comment]
Thanks for the info.
